src/main/java/com/minelittlepony/hdskins/util/net/MoreHttpResponses.java

```diff
@@ -30,7 +25,7 @@
     CloseableHttpResponse getResponse();
 
     default boolean ok() {
-        return getResponseCode() == HttpStatus.SC_OK;
+        return getResponseCode() < 400;
```

Comment:
` getResponseCode() < 300`

Reply:
300 is redirects. Those don't mean error.

Reply:
Depends on your definition of "error". They don't have content, so they're not "ok()".